casr/tests/tests.rs

```diff
@@ -4049,10 +4049,6 @@ fn test_casr_libfuzzer() {
 #[test]
 #[cfg(target_arch = "x86_64")]
 fn test_casr_libfuzzer_atheris() {
-    if lsb_release::info().unwrap().version == "22.04" {
```

Comment:
We may now remove lsb_release from Cargo.toml

Reply:
done